src/.github/workflows/pre-commit.yml.jinja

```
@@ -11,7 +11,11 @@ on:
 
 jobs:
   pre-commit:
+{%- if odoo_version < 14 %}
+    runs-on: ubuntu-20.04
+{%- else %}
     runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```

Let's pin now so we don't have to do it in 2 years?